packages/manager/modules/pci/src/components/project/billing/snapshot-list/index.js

```diff
@@ -0,0 +1,11 @@
+import angular from 'angular';
+
+import component from './component';
+
+const moduleName = 'ovhManagerPciComponentsProjectBillingSnapshotList';
+
+angular
+  .module(moduleName, [])
```

Comment:
Seems to have a dependency to `ui.bootstrap` right? `data-uib-tooltip` for instance.